common/constants/api.js

```diff
@@ -17,14 +17,17 @@ export const createUser = ({ email, password, firstName, lastName, zipcode }) =>
   }).then(({ data }) => data);
 
 export const loginUser = ({ email, password }) =>
-  post('auth/login/', {
-    email,
-    password,
-  }).then(({ data }) => data);
+  post(
+    'auth/login/',
+    {
+      email,
+      password,
+    },
+    { noAuth: true },
```

Comment:
what's `noAuth`?

Reply:
I see now. Maybe: `hasAuthHeaders` for naming.

At the same time, is it really necessary? What's wrong with always setting auth headers for every request?

Reply:
Some funky Django stuff.  It appears that if a request has a JWT in the auth header the very first thing it does it check if it's valid, before it even gets to the view to see if it's an authenticated route or not.

I agree it's kinda weird.  I'm planning on looking into it some more, but it fixed an intermittent issue I was having with stale creds.